migrations/1745877592337_event-observer-timestamps.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable camelcase */
+
+exports.shorthands = undefined;
+
+exports.up = pgm => {
+  pgm.createTable('event_observer_timestamps', {
```

Comment:
Should this new table have a ordered index on the `receive_timestamp` column?

Reply:
I don't think it's necessary because it will contain only 1 row per event type `/new_block` `/mempool_tx` etc. and we will always retrieve all rows at the same time for prometheus, not sorted by timestamp